packages/plans-grid-next/src/components/storage/hooks/use-storage-string-from-feature.ts

```diff
@@ -0,0 +1,91 @@
+import {
+	FEATURE_1GB_STORAGE,
+	FEATURE_6GB_STORAGE,
+	FEATURE_13GB_STORAGE,
+	FEATURE_50GB_STORAGE,
+	FEATURE_200GB_STORAGE,
+	FEATURE_50GB_STORAGE_ADD_ON,
```

Comment:
`NB:`

In the past, we've talked about `FEATURE_50GB_STORAGE_ADD_ON` and `FEATURE_100GB_STORAGE_ADD_ON` being a misnomer. The add-on is _not_ a feature, and at the very least, it should just be `50GB_STORAGE_ADD_ON`. I know there's a lot more to refactor beyond that, but since we're starting to taking a closer look at this part of the code again, let me know if you think renaming might help with any confusion. I'm happy to set up a PR if we'd like.

Reply:
Good thinking @jeyip 

Let's come back to this later and figure out the best way forward. I'm wondering if we can skip a "meaningful" slug for this altogether and just go with how storage add-ons work (a combination of `PRODUCT_1GB_SPACE` & `quantity`). So in a way finding a solution that won't need to enumerate/define the options via slugs. Then for the state store we could just store the index of the item selected, not the slug (and whether add-on).

Reply:
> It's unfortunate that my initial technical implementation of add-ons relying on feature slugs is spreading a bit more 😕

Not at all! I can only feel sympathetic seeing the complexity around this. The problem is deep, and we also didn't exactly have time to "breathe" at the time. 😎